realm/realm-library/src/androidTest/java/io/realm/RealmIntegerTests.java

```diff
@@ -82,8 +83,38 @@ public void basic() {
      * if either arg is an int. That will cause sign extension.
      */
     @Test
-    public void getters() {
-        RealmInteger ri = new RealmInteger(0x5555444433332211L);
+    public void unmanaged_getters() {
```

Comment:
mmm...  sot the finest grained part comes first?  Like writing 01-05-2017?  That means that the names do not sort into categories.